### DISCUSSION TITLE
include in `NixOS/templates`?

### DISCUSSION THREAD

**Discussion by peterbecich** [srid/haskell-flake#131] [Ideas]
May I suggest including `haskell-flake` or `haskell-template` here https://github.com/NixOS/templates?
Then the project would be included in `nix flake show templates`, and it would enable the same thing that `haskell.nix` has:
```
nix flake init --template templates#haskell-nix --impure
```

Thanks, this Flake is great

**Comment by srid** [comment]
Sure; if you want to open a PR (in that repo), go ahead. I'd continue to maintain the [example](https://github.com/srid/haskell-flake/tree/master/example) in this repo for CI tests.
